deploy/cr.yaml

```diff
@@ -608,6 +608,7 @@ spec:
   backup:
 #    allowParallel: true
     image: perconalab/percona-xtradb-cluster-operator:main-pxc8.0-backup
+#    ttlSecondsAfterFinished: 60
```

Comment:
people might understand this as a recommendation and 60 seconds is too short, especially for failed backups. let's make it something like 3600